kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/security/adapters/KeycloakSecurityAdapter.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.services.impl.security.adapters;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.kie.server.api.security.SecurityAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.keycloak.KeycloakPrincipal;
+
+public class KeycloakSecurityAdapter implements SecurityAdapter {
+
+    private static final Logger logger = LoggerFactory.getLogger(KeycloakSecurityAdapter.class);
+
+    private boolean active;
+
+    public KeycloakSecurityAdapter() {
+        try {
+            Class.forName("org.keycloak.KeycloakPrincipal");
```

Comment:
woundlt this be true because of the new keycloak dependencies that have been added in https://github.com/kiegroup/droolsjbpm-integration/pull/1905/files#diff-5609012f707af280a5b3aeebdde69f01R211 ?

Reply:
yes, it is not necessary